src/__tests__/fixtures/schema/search/matrices.ts

```diff
@@ -19,7 +19,7 @@ export class Cell {
 	@SearchField()
 	y: number;
 
-	@SearchField()
+	@SearchField({ id: true })
```

Comment:
Should we throw an error instead?

Reply:
We are not throwing an error for sort and facet attributes as well. So I am following the same strategy.